apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.html

```diff
@@ -1,23 +1,39 @@
-<div (keyup)="onKeyInput($event)">
+<div (keyup)="onKeyInput($event)" class="vo-theme">
   <h1 class="page-subtitle">{{'VO_DETAIL.MEMBERS.TITLE' | translate}}</h1>
-  <button mat-flat-button color="accent" class="mr-2" (click)="onAddMember()">
-    {{'VO_DETAIL.MEMBERS.ADD_MEMBER' | translate}}
-  </button>
-  <button mat-flat-button color="warn" class="mr-2" [disabled]="selection.isEmpty()" (click)="onRemoveMembers()">
-    {{'VO_DETAIL.MEMBERS.REMOVE_MEMBERS' | translate}}
-  </button>
-  <button mat-stroked-button class="mr-2 mb-2" (click)="onListAll()">
-    {{'VO_DETAIL.MEMBERS.LIST_ALL' | translate}}
-  </button>
+    <button mat-flat-button color="accent" class="mr-2" (click)="onAddMember()">
+      {{'VO_DETAIL.MEMBERS.ADD_MEMBER' | translate}}
+    </button>
+    <button mat-flat-button color="warn" class="mr-2" [disabled]="selection.isEmpty()" (click)="onRemoveMembers()">
+      {{'VO_DETAIL.MEMBERS.REMOVE_MEMBERS' | translate}}
+    </button>
+    <button mat-stroked-button class="mr-2 mb-2" (click)="onListAll()">
+      {{'VO_DETAIL.MEMBERS.LIST_ALL' | translate}}
+    </button>
 
-  <mat-form-field class="mr-2 search-field no-border-mat-input">
-    <label>
-      <input matInput autocomplete="false" placeholder="{{'VO_DETAIL.MEMBERS.SEARCH_DESCRIPTION' | translate}}" [(ngModel)]="searchString">
-    </label>
-  </mat-form-field>
-  <button mat-flat-button color="primary" class="mr-2" [disabled]="searchString.length === 0" (click)="onSearchByString()">
-    <mat-icon>search</mat-icon>
-  </button>
+    <mat-form-field class="mr-2 ml-2">
```

Comment:
When putting margin between items on one line, prefer setting margin only on the right side. In case of viewing on a smaller device, it looks weird when the items are not aligned in a single column because of the margin on the left.